controllers/admin_client_mock.go

```diff
@@ -41,9 +41,9 @@ type mockAdminClient struct {
 	Cluster                                  *fdbv1beta2.FoundationDBCluster
 	KubeClient                               client.Client
 	DatabaseConfiguration                    *fdbv1beta2.DatabaseConfiguration
-	ExcludedAddresses                        []string
+	ExcludedAddresses                        map[string]struct{}
```

Comment:
```suggestion
	ExcludedAddresses                        map[string]fdbv1beta2.None
```

We have a wrapper around that empty struct https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/api/v1beta2/none.go#L23-L25 to represent that this means it's a "None". We should use this everywhere, there might be some places where we missed to refactor that.

Reply:
fixed